src/platform/platform-client.ts

```diff
@@ -174,6 +173,10 @@ export class PlatformClient extends Client {
         case 403:
           customError = new ForbiddenError(`${error.response.status}: ${JSON.stringify(error.response.data)}`);
           break;
+
+        case 409:
+          customError = new Error(`${JSON.stringify(error.response.data)}`);
+          break;
       
         default:
           customError = new Error(`Generic error: ${JSON.stringify(error.response.data)}`);
```

Comment:
What's your idea here? These are not generic errors anymore. With your implementation in account-api the errors thrown now is.
- UnauthorizedException
- BadRequestException
- NotFoundException
- ConflictException
- Probably more will be added

Should we remove the switch case all together?

Reply:
fixed. forgot to remove this switch upon last account api fixes